packages/passport/sdk/src/zkEvm/walletHelpers.ts

```diff
@@ -142,12 +143,50 @@ export const getSignedMetaTransactions = async (
   ]);
 };
 
-const decodeRelayerTypedDataSignature = (relayerSignature: string) => {
+const decodeRelayerSignature = (relayerSignature: string): UnrecoveredSignature => {
   const signatureWithThreshold = `0x0000${relayerSignature}`;
   return sequenceCoreV1.signature.decodeSignature(signatureWithThreshold);
 };
 
-export const getSignedTypedData = async (
+export const packSignatures = (
+  EOASignature: string,
+  EOAAddress: string,
+  relayerSignature: string,
+): string => {
+  const signedDigest = `${EOASignature}${ETH_SIGN_FLAG}`;
+
+  // Combine the relayer and user signatures; sort by address to match the imageHash order
+  const { signers: relayerSigners } = decodeRelayerSignature(relayerSignature);
+  const combinedSigners = [
+    ...relayerSigners,
+    {
+      isDynamic: false,
+      unrecovered: true,
+      weight: SIGNATURE_WEIGHT,
+      signature: signedDigest,
+      address: EOAAddress,
+    },
+  ];
+  const sortedSigners = combinedSigners.sort((a, b) => {
+    const bigA = BigNumber.from(a.address);
+    const bigB = BigNumber.from(b.address);
+
+    if (bigA.lte(bigB)) {
+      return -1;
+    } if (bigA.eq(bigB)) {
+      return 0;
+    }
+    return 1;
+  });
+
+  return sequenceCoreV1.signature.encodeSignature({
+    version: 1,
+    threshold: EIP712_SIGNATURE_THRESHOLD,
```

Comment:
can we rename EIP712_SIGNATURE_THRESHOLD, since it is for both eip721 and erc191